operator/config/samples/lifecycle_v1alpha3_keptntaskdefinition_container.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnTaskDefinition
+metadata:
+  name: keptntaskdefinition-sample-container
+spec:
+  container:
+    name: testy-test
+    image: busybox:1.36.0
+    command:
+      - 'sh'
+      - '-c'
+      - 'sleep 30'
```

Comment:
I know this is mostly one on one with k8s spec so it is not rocket science to fill it in, but maybe we could add the things that are different for us here in the example, for instance how to set up the volume mount? maybe using it inside busybox to print something? Also it would be precious for documentation later if the two examples uses different containers.